src/main/scala/cromwell/binding/values/WdlFileLike.scala

```diff
@@ -0,0 +1,10 @@
+package cromwell.binding.values
+
+import cromwell.binding.types.WdlFileType
+
+/**
+ * WDL value which identifies the location of a File-like object.
+ */
+trait WdlFileLike extends WdlPrimitive {
+  val wdlType = WdlFileType
```

Comment:
I brought this up in the meeting earlier, but this line seemed kinda wrong to me... I'm not sure if this value is used anywhere or if this is just a default value.  Maybe this needs a new type `WdlFileLikeType` (yuck...)